satellite/satellitedb/migrate.go

```diff
@@ -719,6 +719,18 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 					`ALTER TABLE projects ADD usage_limit bigint NOT NULL DEFAULT 0;`,
 				},
 			},
+			{
+				Description: "Add invitee_credit_in_gb and award_credit_in_gb columns, delete type and credit_in_cents columns",
+				Version:     25,
+				Action: migrate.SQL{
+					`DELETE FROM offers WHERE id = 1;`,
```

Comment:
I feel like there's a better way to delete the data that was there before, but I'm not sure what that would be..